File: 指纹识别类似支付宝/TouchLock/FirstViewController.m
//
//  FirstViewController.m
//  TouchLock
//
//  Created by dai.fy on 15/4/30.
//  Copyright (c) 2015年 dai.fy. All rights reserved.
//

#import "FirstViewController.h"
#import "SSKeychain.h"
#import "VENTouchLock.h"
#import "ViewController.h"
@interface FirstViewController ()

@end

@implementation FirstViewController



- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view from its nib.
    BOOL isSet = [[VENTouchLock sharedInstance] isPasscodeSet];
    if (isSet) {
        MainViewController *vc = [[MainViewController alloc] init];
        vc.didFinishWithSuccess = ^(BOOL success, VENTouchLockSplashViewControllerUnlockType unlockType){
            
        };
        [self presentViewController:vc animated:NO completion:^{
            
        }];
    }else{
//        [self presentViewController:[[ViewController alloc] init] animated:YES completion:^{
//            
//        }];
        [[UIApplication sharedApplication].windows[0] setRootViewController:[[ViewController alloc] init]];
    }
}

- (IBAction)clearKeychain:(id)sender {
    [SSKeychain deletePasswordForService:@"TouchLock" account:[[NSUserDefaults standardUserDefaults] objectForKey:@"account"]];
    
}
@end

File: ViewWidget/ViewWidget/FirstViewController.swift
//
//  FirstViewController.swift
//  ViewWidget
//
//  Created by DaiFengyi on 16/1/21.
//  Copyright © 2016年 DaiFengyi. All rights reserved.
//

import UIKit

class FirstViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        let banner = FYScrollBanner(frame: CGRectMake(0, 0, 320, 200), index: 0)
        banner.imageList = [UIImage(named: "1-1")!, UIImage(named: "1-2")!, UIImage(named: "1-3")!, UIImage(named: "1-4")!]
        self.view.addSubview(banner)
    }
    

}

